src/core/tsi/alts/zero_copy_frame_protector/alts_zero_copy_grpc_protector.cc

```diff
@@ -218,6 +218,23 @@ static tsi_result alts_zero_copy_grpc_protector_unprotect(
   return TSI_OK;
 }
 
+static tsi_result alts_zero_copy_grpc_protector_unprotect_get_frame_size(
+    tsi_zero_copy_grpc_protector* self, grpc_slice_buffer* protected_slices,
+    grpc_slice_buffer* unprotected_slices, int* last_incomplete_frame_size) {
+  if (self == nullptr || unprotected_slices == nullptr ||
```

Comment:
alts_zero_copy_grpc_protector_unprotect already returns INVALID_ARGUMENT if self, protected_slices, or unprotected_slices is nullptr so you shouldn't need to check them here too

Reply:
Fixed.